test/stb_image_test.exs

```diff
@@ -58,4 +58,21 @@ defmodule StbImageTest do
     assert frames ==
              [<<180, 128, 70, 255, 171, 119>>, <<61, 255, 65, 143, 117, 255>>]
   end
+
+
+  test "save image png" do
+    read = StbImage.from_file(Path.join(__DIR__, "test.png"))
+    {:ok, img, {height, width, num_channels}, _, _} = read
+    :ok = StbImage.to_file(Path.join(__DIR__, "save_test.png"), "png", img, width, height, num_channels)
```

Comment:
I think save_test.tga and save_test.png should be added to .gitignore

Reply:
Also, wouldn't it be better to keep test.png in the priv_dir?

Reply:
same for test.bmp and test.tga

Reply:
No, test stuff should not be in priv because priv would be part of a release and similar.